Tasks/BelkevichAleksandr/Task3/task3.py

```diff
@@ -0,0 +1,103 @@
+def selection():
+    '''
+    Function for checking selection input from user.
+
+    '''
+    while True:
+        try:
+            number = int(input("Please enter length line (minimum 36 maximum 70 characters): "))
+
+            # check if the entered number is correct
+            if 36 <= number <= 70:
+                return number
+
+            # output for explanation
+            print(f"Error, {number} isn't within the specified range")
+            print("The specified range line = min 36 and max 70 characters")
+        except ValueError:
+            print("ValueError, not an integer entered")
+            continue
+
+
+def converter_text_in_lst(lst):
+    '''
+    Function to sort text from a file by a specified length of lines from the user in list for list.
+
+    '''
+    lst_lines = []
+    
+    # splitting a string into words
+    mas_line = lst.split(" ")
+
+    lst_word = []
+    sum_mas_string = 0
+    count = 0
+
+    # iterating word from lines
+    for word in mas_line:
+        # notation to check string length with word and space
+        sum_mas_string += len(word) + 1
+        # variable to check the last word in the line
+        count += 1
+
+        if sum_mas_string < string_len and count == len(mas_line):
+            lst_word.append(f"{word}\n")
+            lst_lines.append("".join(lst_word))
+        elif sum_mas_string < string_len:
+            lst_word.append(word)
+            lst_word.append(" ")
+        else:
+            del lst_word[-1::]
+            sum_mas_string -= len(word) + 2 # 2 is the two spaces between the last word in the sum_mas_string
+            lst_lines.append(correct_line_for_len_string(lst_word, sum_mas_string))
+            lst_lines.append("\n")
+            lst_word = []
+            sum_mas_string = len(word) + 1
+            lst_word.append(word)
+            lst_word.append(" ")
+
+    #Question? I can delete not needed variables here or not
+    #del lst_word
+    #del sum_mas_string
+    #del count
```

Comment:
you could, but there is not much sense in it, the garbage collector will take care of these variables